### PR TITLE
scope_based_token_issuance_policy_template added

### PR DESCRIPTION
**Pull request by Buddhimah**
**Related Issues**

> wso2/product-is#5358
> wso2/product-is#5804

### CODE REVIEW

File: features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/scope_based_token_issuance_policy_template.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<Policy xmlns="urn:oasis:names:tc:xacml:3.0:core:schema:wd-17"  PolicyId="scope_based_token_issuance_policy_template" RuleCombiningAlgId="urn:oasis:names:tc:xacml:1.0:rule-combining-algorithm:first-applicable" Version="1.0">
+    <Description>This policy template provides ability to validate OAuth2 access token to a given service provider(defined by SP_NAME) in the validation flow based on the scope (defined by SCOPE_1). It allows validation of token on time of the day (eg. between 09:00:00 to 18:00:00).</Description>
```
Comment (senthalan) [inline features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/scope_based_token_issuance_policy_template.xml]: can we change the description, it says 'validation flow'

File: features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/p2.inf [hunk 2]
```diff
@@ -6,3 +6,4 @@ org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../reso
 org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../resources/identity/policies/xacml/); \
 org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../resources/identity/policies/xacml/default); \
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.identity.oauth2.validators.xacml.server_${feature.version}/scope_based_token_validation_policy_template.xml,target:${installFolder}/../../resources/identity/policies/xacml/default/scope_based_token_validation_policy_template.xml,overwrite:true);\
+org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.identity.oauth2.validators.xacml.server_${feature.version}/scope_based_token_issuance_policy_template.xml,target:${installFolder}/../../resources/identity/policies/xacml/default/scope_based_token_issuance_policy_template.xml,overwrite:true);\
```
Comment (senthalan) [inline features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/p2.inf]: add new line

File: features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/scope_based_token_issuance_policy_template.xml (hunk 1 above)
Comment (Buddhimah) [reply features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/scope_based_token_issuance_policy_template.xml]: done

File: features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/p2.inf (hunk 2 above)
Comment (Buddhimah) [reply features/org.wso2.carbon.identity.oauth2.validators.xacml.server.feature/resources/p2.inf]: done

**Pull request merged**